Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs

```diff
@@ -22,6 +22,7 @@ public class ModelOverrider : MonoBehaviour
     {
         const string k_CommandLineModelOverrideFlag = "--mlagents-override-model";
         const string k_CommandLineQuitAfterEpisodesFlag = "--mlagents-quit-after-episodes";
+        const string k_CommandLineQuitOnLoadFailure = "--mlagents-quit-on-load-failure";
```

Comment:
@xiaomaogy you'd want to add this to the commandline args

Reply:
This is perfect.